modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -7,29 +7,23 @@
 
-NOTE: When the certificates used by {prod-short} components or by an external service are signed by an untrusted CA, it can be necessary to import the CA certificate in the {prod-short} installation so that every {prod-short} component will consider them as signed by a trusted CA.
-
-Typical cases that may require this addition are:
-
-* When the underlying {orch-name} cluster that uses TLS certificates signed by a CA that is not trusted.
-* When {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
+When the certificates used by {prod-short} components or by an external service are signed by an untrusted CA, it's necessary to import the CA certificate in the {prod-short} installation so that every {prod-short} component considers them as signed by a trusted CA. Following cases typically require this addition:
+* The underlying {orch-name} cluster uses TLS certificates signed by a CA that isn't trusted.
+* {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
 
 {prod-short} uses labeled ConfigMaps in {orch-namespace} as sources for TLS certificates. The ConfigMaps can have an arbitrary number of keys with a random number of certificates each.
 
 [NOTE]
 ====
-When the OpenShift cluster contains cluster-wide trusted CA certificates added through the link:https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration], {prod-short} Operator detects them and automatically injects them into this ConfigMap:
-
-- {prod-short} automatically labels the ConfigMap with the `config.openshift.io/inject-trusted-cabundle="true"` label.
-- Based on this annotation, OpenShift automatically injects the cluster-wide trusted CA certificates inside the `ca-bundle.crt` key of ConfigMap
+When an OpenShift cluster contains cluster-wide trusted CA certificates added through the link:https://docs.openshift.com/container-platform/4.10/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration], {prod-short} Operator detects them and automatically injects them into a ConfigMap. {prod-short} automatically labels the ConfigMap with the `config.openshift.io/inject-trusted-cabundle="true"` label. Based on this annotation, OpenShift automatically injects the cluster-wide trusted CA certificates inside the `ca-bundle.crt` key of ConfigMap.
```

Comment:
Usability 

I would remove the link. It distracts the user from the task at hand and prompts the user to leave your documentation before even finishing the sentence. 

I think links are best used in prereqs or additional resources. They're too distracting in a procedure or informative paragraph, etc 

`When an OpenShift cluster contains cluster-wide trusted CA certificates added through the cluster-wide-proxy configuration, {prod-short} Operator detects the certificates and automatically injects them into a ConfigMap.`

Reply:
@jc-berger Thank you for going the extra mile on this. I agree that adding the info to Additional resources could be helpful, too. But I don't agree with removing it from the text. Links are an inherent and useful part of the web. It would be a different matter if, let's say, every second term in the paragraph was a link, or if the link was overly long. But that's not the case. This is well placed and useful. Users are well aware of hypertext's capability to include links, and they can use them when they need. It's not confusing.

Btw, see pg. 270 of the IBM SG.